### PR TITLE
Continuation of PR#2834

### PR DESCRIPTION
**Pull request by ahardin-rh**
Continuation of https://github.com/openshift/openshift-docs/pull/2834

### CODE REVIEW

**Comment by ahardin-rh** [comment]
cc @adellape

**Comment by ahardin-rh** [comment]
Build of revisions: http://file.rdu.redhat.com/~ahardin/09212016/backup-restore/admin_guide/backup_restore.html

File: admin_guide/backup_restore.adoc [hunk 1]
```diff
@@ -191,3 +220,168 @@ namespaces and projects):
...
+== Backing Up Other Resources
+
+This section reviews how to back up resources like role bindings, service
+accounts, secrets, and persistent volume claims.
+
+[[backup-rolebindings]]
+=== Role Bindings
+
+Sometimes custom policy
+xref:../admin_guide/manage_authorization_policy.html#managing-role-bindings[role
```
Comment (adellape) [inline admin_guide/backup_restore.adoc]: s/html/adoc/

File: admin_guide/backup_restore.adoc [hunk 2]
```diff
@@ -191,3 +220,168 @@ namespaces and projects):
...
++
+Alternatively, you can scale down the deployment to 0, and then up again:
++
+----
+$ oc scale --replicas=0 dc/jenkins
+$ oc scale --replicas=1 dc/jenkins
+----
+
+[[backup-additional-resources]]
+== Backing Up Other Resources
```
Comment (adellape) [inline admin_guide/backup_restore.adoc]: As mentioned on IRC, maybe these individual objects could be subsections under "Project Backup"?

**Review by adellape — CHANGES_REQUESTED** [review]
1 bad link + 1 suggestion. LGTM otherwise!

File: admin_guide/backup_restore.adoc (hunk 1 above)
Comment (ahardin-rh) [inline admin_guide/backup_restore.adoc]: Thanks for catching that! :scream_cat:

**Review by adellape — APPROVED** [review]

**Comment by ahardin-rh** [comment]
[rev_history]
|xref:../admin_guide/backup_restore.adoc#admin-guide-backup-and-restore[Backup and Restore]
|Added Prerequisites details and created new sections for Application Data Backup, Application Data Restore, Project Restore, as well as backing up Role Bindings, Service Accounts, Secrets, and Persistent Volume Claims.
%

**Pull request merged**